contracts/script/DeployMockAvsRegistries.s.sol

```diff
@@ -0,0 +1,305 @@
+                    addressConfig.communityMultisig,
+                    addressConfig.churner,
+                    addressConfig.ejector,
+                    addressConfig.pauser,
+                    0, // 0 initialPausedStatus means everything unpaused
+                    quorumsOperatorSetParams,
+                    quorumsMinimumStake,
+                    quorumsStrategyParams
+                )
+            );
+        }
+
+        require(
+            Ownable(address(registryCoordinator)).owner() != address(0),
+            "Owner uninitialized"
+        );
+
+        // WRITE JSON DATA
+        {
+            string memory parent_object = "parent object";
+            string memory deployed_addresses = "addresses";
+            vm.serializeAddress(
+                deployed_addresses,
+                "proxyAdmin",
+                address(mockAvsProxyAdmin)
+            );
+            vm.serializeAddress(
+                deployed_addresses,
+                "mockAvsServiceManager",
+                address(mockAvsServiceManager)
+            );
+            vm.serializeAddress(
+                deployed_addresses,
+                "mockAvsServiceManagerImplementation",
+                address(mockAvsServiceManagerImplementation)
+            );
+            vm.serializeAddress(
+                deployed_addresses,
+                "registryCoordinator",
+                address(registryCoordinator)
+            );
+            vm.serializeAddress(
+                deployed_addresses,
+                "registryCoordinatorImplementation",
+                address(registryCoordinatorImplementation)
+            );
+            string memory deployed_addresses_output = vm.serializeAddress(
+                deployed_addresses,
+                "operatorStateRetriever",
+                address(operatorStateRetriever)
+            );
+
+            // serialize all the data
+            string memory finalJson = vm.serializeString(
+                parent_object,
+                deployed_addresses,
+                deployed_addresses_output
+            );
+
+            writeOutput(finalJson, "mockavs_deployment_output");
+        }
+        return
+            MockAvsContracts(
+                mockAvsServiceManager,
+                registryCoordinator,
+                operatorStateRetriever
+            );
+    }
+
+    function _writeContractsToRegistry(
+        ContractsRegistry contractsRegistry,
+        EigenlayerContracts memory eigenlayerContracts,
+        MockAvsContracts memory mockAvsContracts
+    ) internal {
+        contractsRegistry.registerContract(
+            "mockAvsServiceManager",
+            address(mockAvsContracts.mockAvsServiceManager)
+        );
+        contractsRegistry.registerContract(
+            "mockAvsRegistryCoordinator",
+            address(mockAvsContracts.registryCoordinator)
+        );
+        contractsRegistry.registerContract(
+            "mockAvsOperatorStateRetriever",
```

Comment:
this is where we populate the ContractsRegistry during deployment, so that we can get the addresses during the tests